README.md

```diff
@@ -1,76 +1,32 @@
-
-For help or support:
-
-- read our [documentation](/docs)
-- raise an [issue](https://github.com/alphagov/gsp/issues)
-- message the team on the Reliability [Engineering Slack channel](https://gds.slack.com/messages/CAD6NP598) [#reliability-eng](https://gds.slack.com/messages/CAD6NP598)
-
-
+GSP ([GDS](https://www.gov.uk/government/organisations/government-digital-service) Supported Platform) was a Kubernetes distribution based on Amazon EKS.
+
+Technically:
+* The Kubernetes/EKS version is behind - we're on 1.16 and the latest is 1.20. 1.16 will not be possible to use after July 2021.
+ * There's a TODO in `pipelines/deployer/deployer.yaml` about k8s 1.15 we can probably remove.
+* GSP relies on Istio 1.5.8 which became end-of-life on 2020-08-24. Also 1.6 was end-of-life on 2020-11-23, 1.7 was end-of-life on 2021-02-25, and 1.8 will be end-of-life on 2021-05-12.
+* GSP ran Prometheus and Grafana through prometheus-operator 8.15.6, and it's no longer developed at `git@github.com:helm/charts.git` stable/prometheus-operator - the latest version of chart we were using is now deprecated.
+* The check-vulnerabilities job in each cluster deployment pipeline would find all sorts of things in the third-party images we used like cluster-autoscaler, concourse-web, external-dns, fluentd-cloudwatch, fluentd-kubernetes-daemonset, and more. Some of these may be resolvable by upgrading the version of the software used.
+* It's based on Terraform but with some weird extra CloudFormation that should be merged into the Terraform - `modules/k8s-cluster/data/nodegroup-v2.yaml` and (especially obscure, but possibly unnecessary depending on the item below) `modules/k8s-cluster/data/nodegroup.yaml`.
+* We're not completely certain that the cluster-management nodes still necessary, it may be possible to put gatekeeper and the cluster-autoscaler on normal worker nodes.
+* There is a strange distinction between the k8s-cluster and gsp-cluster terraform modules which should probably be eliminated.
+* Some of the docs are written with gds-cli in mind but gds-cli dropped support for GSP in version 4, so you'll either need to convince gds-cli to reimplement that support or eliminate the dependency by going back to plain aws-vault.
+* We wrote our own service operator, but then AWS made https://aws.amazon.com/blogs/containers/aws-controllers-for-kubernetes-ack/ which probably obsoletes part of it already - and in future, likely all of it. If you're going to re-use the GSP code, remove the obsolete parts of our service operator (ECR, SQS, S3) and use AWS's, then eventually remove ours all together when they add RDS+ElastiCache
+* We tried to have working, daily replacement of the underlying EC2 instances, but our experience with node rolling in GSP was complicated, with multiple incidents. There's some problems lurking somewhere that need tracking down and solving. It may be possible to use EKS Fargate and eliminate the EC2 instances.
```

Comment:
```suggestion
* We tried to have working, daily replacement of the underlying EC2 instances, but our experience with node rolling in GSP was complicated, with multiple incidents. There are some problems lurking that need tracking down and solving. It may be possible to use EKS Fargate and eliminate the EC2 instances.
```